.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: npm
```

Comment:
I'm not convinced. All npm dependencies in this repo are exclusively for the build and test toolchain. There's only really a need to upgrade whenever we're looking into releasing a new version of the library, which happens very rarely. I think this will create quite a bit of noise at no clear benefit.

I think even running this monthly would create an amount of noise that I'm personally currently not willing to handle.

Reply:
Do you want me to remove that one, but leave the GitHub Actions part?